src/main/java/com/github/manolo8/darkbot/core/manager/GroupManager.java

```diff
@@ -218,4 +218,68 @@ public int idx(boolean leader) {
         }
     }
 
+    @Override
+    public boolean hasGroup() {
+        return group.isValid();
+    }
+
+    @Override
+    public int getId() {
+        return group.id;
+    }
+
+    @Override
+    public int getSize() {
+        return group.size;
+    }
+
+    @Override
+    public int getMaxSize() {
+        return group.maxSize;
+    }
+
+    @Override
+    public boolean isOpen() {
+        return group.isOpen;
+    }
+
+    @Override
+    public boolean isLeader() {
+        return group.isLeader;
+    }
+
+    @Override
+    public List<? extends eu.darkbot.api.objects.group.GroupMember> getMembers() {
+        return group.members;
+    }
+
+    @Override
+    public eu.darkbot.api.objects.group.@Nullable GroupMember getSelectedMember() {
+        return group.selectedMember;
+    }
+
+    @Override
+    public eu.darkbot.api.objects.group.@Nullable GroupMember getMember(int id) {
+        return group.getMember(id);
+    }
+
+    @Override
+    public void kickMember(int id) {
+        kick(id);
+    }
+
+    @Override
+    public List<? extends eu.darkbot.api.objects.group.GroupMember.Invite> getInvites() {
+        return invites;
+    }
+
+    @Override
+    public void openInvites() {
```

Comment:
This api method alongside a few others in group api make very little sense. This isn't "opening invites" this is just "enforcing the setting to set invites to open or closed", which is too reliant on the settings themselves.

Should consider removing this method from the api, have it be an implementation detail. 

Because of how "independent" the group manager works on its own (it reads settings and invites ppl with the tag, or leaves group, ETC) it maybe makes more sense for the API methods to be focused arround those settings more than acting on the group itself with "sync" methods, ie: maybe makes more sense to add a method that says "accept invite from X player whenever it appears" rather than a method to accept it directly that needs to be called in the precise moment that it is sent